src/util/bit_util.rs

```diff
@@ -392,7 +393,7 @@ impl BitReader {
   /// Gets the current byte offset
   #[inline]
   pub fn get_byte_offset(&self) -> usize {
-    self.byte_offset + self.bit_offset / 8 + 1
+    self.byte_offset + ceil(self.bit_offset as i64, 8) as usize
```

Comment:
@sunchao you are right, we should use ceil here instead of `+1`, this covers case when bit offset is 0. I added tests showing the actual offset for each value we get.